app/helpers/application_helper.rb

```diff
@@ -22,4 +22,9 @@ def ethnicity_options
       ['Middle Eastern', 4]
     ], get_pre_selected_option("ethnicity", nil))
   end
+  
+  def age(dob)
+    now = Time.now.utc.to_date
+    now.year - dob.year - ((now.month > dob.month || (now.month == dob.month && now.day >= dob.day)) ? 0 : 1)
+  end
```

Comment:
This could be put in your model. It calculates the age of a profile based on the current time, using data from that model. Doesn't `now.year - dob.year` work on it's own?